<robot xmlns:xacro="http://www.ros.org/wiki/xacro" name="assy_part_11">
  <xacro:macro name="assy_part_11" params="prefix parent *origin">
    <link name="${prefix}assy_part_11">
      <visual>
        <origin rpy="1.5707 0 0" xyz="0 0 0"/>
        <geometry>
          <mesh filename="package://o2as_parts_description/meshes/11_MBRAC60-2-10.dae" scale="0.001 0.001 0.001"/>
        </geometry>
        <material name="white">
          <color rgba="1 1 1 1"/>
        </material>
      </visual>
      <collision>
        <origin rpy="1.5707 0 0" xyz="0 0 0"/>
        <geometry>
          <mesh filename="package://o2as_parts_description/meshes/11_MBRAC60-2-10.vhacd.dae" scale="0.001 0.001 0.001"/>
        </geometry>
      </collision>
    </link>

    <joint name="${prefix}assy_part_11_joint" type="fixed">
      <parent link="${parent}"/>
      <child link="${prefix}assy_part_11"/>
      <xacro:insert_block name="origin"/>
    </joint>
  </xacro:macro>
</robot>
